### DISCUSSION TITLE
No live preview in main menu of DIW7022 anymore after moving to UCG Max

### DISCUSSION THREAD

**Discussion by Traxxiz** [fabianishere/udm-iptv#357] [Q&A] [answered]
I recently moved from an USG to the new UCG Max to use icw KPN. Installation using the scripts via SSH went smooth and KPN TV is running (almost) fine on my KPN DIW7022 boxes. I use a separate VLAN for IPTV (see screenshot below). The UCG Max works fine, including 2 Unifi camera's connected (also on a separate VLAN).

However, before switching the to UCG Max, the main menu of  the DIW7022 always showed a live preview of the selected channel in the upper right. This is gone, now only after a few seconds, a static picture is shown. Also zapping to other channels takes seconds, while it worked smooth in the past.
Do I need to make a config change? I tried multiple multi-cast settings without success. Attached my current multi-cast setting.

![Multicast config](https://github.com/user-attachments/assets/85fa5a2a-0163-4d7d-b151-d800e881a8b2)
![HomeNet-IPTV](https://github.com/user-attachments/assets/181be927-3293-4f63-a9c8-59305961f521)

**Comment by MrNamelezz** [comment]
I am also noticing this with my new UDM pro . Have you maybe found the reason yet?

**Reply by Traxxiz** [reply]
Unfortunately not, I hoped that someone in the community knows what to do. 
Now we have at least two that like to understand if this can be prevented by config :-)

This week a new version of the KPN box will be released. Although I doubt it is the box (worked fine before moving to UCG), let's see if it brings an improvement related to this topic.

**Reply by dmutsaers** [reply, marked as answer]
I assume you need to edit the config file manually.

https://github.com/fabianishere/udm-iptv/issues/351
https://github.com/fabianishere/udm-iptv/discussions/320
https://github.com/fabianishere/udm-iptv/issues/338

**Reply by MrNamelezz** [reply]
Disabling IGMP proxy and reconfiguring UDM-IPTV works. 

@Traxxiz Maybe you need to check the WAN port or reinstall the UDM-IPTV service. 
I have my previews back and changing the channel is relatively fast now.

**Comment by dmutsaers** [comment]
And make sure IGMP Proxy has been disabled:

![image](https://github.com/user-attachments/assets/fe485a19-ee59-4183-af30-b2d57b4fc6f7)

**Comment by Traxxiz** [comment]
Thanks guys for your support.
Changing the default value for IPTV_WAN_INTERFACE to eth4 and disabling the IGMP-Proxy did the job. 
Happy again :-)
